### PR TITLE
CVE-2019-10173

### PR DESCRIPTION
**Pull request by lpardoRH**

### CODE REVIEW

**Comment by iamleot** [comment]
Hello Laura,
can you please add in the description that version 1.4.11 was fixed,
e.g. by appending at the end:

 `This was fixed in version 1.4.11.'


Thank you!

File: 2019/10xxx/CVE-2019-10173.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
                                 "version": {
                                     "version_data": [
                                         {
-                                            "version_value": "1.4.11"
+                                            "version_value": "fixed in 1.4.11"
```
Comment (iamleot) [inline 2019/10xxx/CVE-2019-10173.json]: Should it be adjusted to `from 1.4.10 to 1.4.11`? (only 1.4.10 seems affected according upstream)

File: 2019/10xxx/CVE-2019-10173.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "It was found that xstream API version 1.4.10 introduced a regression for a previous deserialization flaw. If the security framework has not been initialized, it may allow a remote attacker to run arbitrary shell commands when unmarshalling XML or any supported format. e.g. JSON. (regression of CVE-2013-7285)"
+                "value": "It was found that xstream API version before 1.4.11 introduced a regression for a previous deserialization flaw. If the security framework has not been initialized, it may allow a remote attacker to run arbitrary shell commands when unmarshalling XML or any supported format. e.g. JSON. (regression of CVE-2013-7285)"
```
Comment (iamleot) [inline 2019/10xxx/CVE-2019-10173.json]: If I understand correctly only 1.4.10 version is affected, what about changing it from:

 `stream API version before 1.4.11`

to:

  `stream API version 1.4.10 before 1.4.11`?

Comment (lpardoRH) [reply 2019/10xxx/CVE-2019-10173.json]: you are right! Thanks for pointing this out

**Comment by iamleot** [comment]
Thank you!

**Pull request merged**